python/fledge/common/acl_manager.py

```diff
@@ -27,27 +38,44 @@ async def _notify_service_about_acl_change(self, entity_name, acl, reason):
             service = services[0]
         except DoesNotExist:  # Does not exist
             _logger.error("Cannot notify the service {} "
-                          "about {}. It does not exist.".format(entity_name, reason))
+                          "about {}. It does not exist in service registry.".format(entity_name, reason))
+            _logger.info("Moved {} to pending. And pending is {}".format(entity_name,
+                                                                         self._pending_notifications))
+
+            self._pending_notifications[entity_name] = acl
```

Comment:
if it does not exist in registry there is nothing as pending.
Even not for shutdown state